atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/CleanTransactionRange.java

```diff
@@ -76,7 +74,7 @@ protected int executeTimestampCommand(AtlasDbServices services) {
             }
 
             long commitTs = TransactionConstants.getTimestampForValue(value.getContents());
-            if (commitTs <= timestamp) {
+            if (commitTs <= backupTimestamp) {
```

Comment:
I dont' think that actually makes sense. If anything it relates more to being a backupCommitTimestamp "filter". Either way I don't think it fits into either name and should just abstractly be "backupTimestamp"